### ISSUE TITLE
support IndependentOperator in openmc.deplete calls

### ISSUE THREAD

**Issue by shimwell** [openmc-dev/openmc#2668]
## Description
Currently the ```openmc.deplete method``` makes use of the ```CoupledOperator``` class.

For fusion with relatively low burnup compared to fission fuel we might be better of with the ```IndependentOperator``` as this will allow us to do long pulse schedules quickly. This speed increase is perhaps particularly useful for ICF fusion where we have lots of short pulses compared to MCF which has less but longer pulses.

Being able to specific the IndependentOperator for deplete simulations is marked as aTODO in the openmc source code
https://github.com/paulromano/openmc/blob/83c946f554725596818a156ec2a03b4fb708ee07/openmc/model/model.py#L399

However I don't see this task in the [Shutdown Dose Rate Workflow plan](https://github.com/orgs/openmc-dev/projects/1/views/1), I was wondering if this is something anyone is already planning on tackling this or if it is unassigned?

## Alternatives
leave the code as it is and accept the slower run times

## Compatibility
add to existing args for openmc.deplete, but not change default behaviour

**Comment by paulromano** [comment]
@shimwell Yes, we are indeed hoping to get this working with the `Model` class. One step at a time though -- once #2595 is merged, we'll continue trying to streamline things.
